Create concourse pipeline

m-richo opened the pull request:
new pipeline and new s3 bucket for boshrelease

Commit:
initial pipeline to create boshreleases for graylog

Commit:
update object_id for new s3 bosh bucket name. remove old `.final_builds` dir with reference to no longer used objects

gordcorp commented on ci/create-bosh-release.sh:
```diff
@@ -0,0 +1,59 @@
+- name: $RELEASE_NAME
+  version: $VERSION
+  sha1: $RELEASE_SHA1
+  url: https://github.com/govau/${RELEASE_NAME}-boshrelease/releases/download/v${VERSION}/${RELEASE_NAME}-${VERSION}.tgz
```
s/govau/AusDTO. Or should we just move this repo to the govau org?

m-richo replied:
lets move it.  👍

gordcorp commented on ci/create-pipeline.sh:
```diff
@@ -0,0 +1,12 @@
+
+fly validate-pipeline --config pipeline.yml
+
+fly --target ${TARGET} set-pipeline --config pipeline.yml --pipeline create-boshrelease-instant-https -n -l credentials.yml
```
s/create-boshrelease-instant-https/create-boshrelease-graylog/g

m-richo replied:
ah, good spot

Commit:
correct pipeline name to be graylog.
update pipeline resources now that repo has been migrated from `AusDTO` Github org to `govau`
